lib/SourceMapDevToolPlugin.js

```diff
@@ -264,12 +264,24 @@ class SourceMapDevToolPlugin {
 								contentHash: createHash("md4")
 									.update(sourceMapString)
 									.digest("hex")
-							});
+							};
+							let sourceMapFile = compilation.getPath(
+								sourceMapFilename,
+								dataForTemplateIdentifiers
+							);
 							const sourceMapUrl = options.publicPath
 								? options.publicPath + sourceMapFile.replace(/\\/g, "/")
 								: path
 										.relative(path.dirname(file), sourceMapFile)
 										.replace(/\\/g, "/");
+
+							if (currentSourceMappingURLComment !== false) {
```

Comment:
Move the content of this `if` block into the next `if` block since they are equal